.github/workflows/detekt.yml

```diff
@@ -0,0 +1,62 @@
+# This is a basic workflow that is manually triggered
+
+name: Detekt 
+
+# Controls when the action will run. Workflow runs when manually triggered using the UI
+# or API.
+on:
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+      name:
+        # Friendly description to be shown in the UI instead of 'name'
+        description: 'Detekt'
+        # Default value if no value is explicitly provided
+        default: 'World'
+        # Input has to be provided for the workflow to run
+        required: true
+        # The data type of the input
+        type: string
+  
+  push:
+    branches:
+      - feature/detekt
```

Comment:
We'll need to review this before merging, and probably remove it

Reply:
This is totally intended at this point. That makes detekt to execute only in the current branch because the work is not finished. This will not be removed but replaced for a correct policy that we will agree in.